test_elzzz

```diff
@@ -0,0 +1,5 @@
+dit is ook een test file van de andere Els ... 
+
+Ik ben Mega mindy, Mega mindy
+'K ben een echte superheld
+Die elke boef op aarde velt
```

Comment:
Maar de vraag is of dit wel gender-balanced is. Wekt 'boef' niet sowieso het beeld van een man op? Mag ik suggereren: 'delinquent'? Dat voelt gender-neutraler aan.

Reply:
zeer goede suggestie!